server/sockets/events/Investments.js

```diff
@@ -0,0 +1,5 @@
+exports.receiveInvestment = (io, socket) => ({ id, name, projectName }) => {
+  socket.join(id);
+  console.log(`${name} invested in your project, ${projectName}!`);
```

Comment:
You probably don't need this console log here

Reply:
I will keep it for debugging purposes though, similar to how we have it console log when it sockets are connected and disconnected